rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/activity/CovidLoginActivity.java

```diff
@@ -57,4 +59,10 @@ protected Class getHomeActivityClass() {
     protected String getRegisterTableName() {
         return COVID_PATIENTS;
     }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        JobManager.instance().cancelAllForTag(RDTSyncSettingsServiceJob.TAG);
```

Comment:
This just stops settings sync, does it stop regular sync as well?

Reply:
According to doc: Cancel all pending JobRequests and running jobs.

Reply:
yes, but for this tag, `RDTSyncSettingsServiceJob.TAG`, right? shouldn't we cancel all running jobs?

Reply:
yeah i have a method for cancel all jobs but do i really need to cancel all jobs?

Reply:
why do we need them running after logout? especially the sync job which keeps checking authorization and syncing data.

Reply:
now canceled all running and pending jobs on logout.